exercises/nucleotide-count/canonical-data.json

```diff
@@ -17,7 +24,32 @@
           }
         },
         {
-          "description": "strand with repeated nucleotide",
+          "comments" : [
+            "Should check for null-references and assume an empty input."
+          ],
+          "description": "returns zeroes for an undefined/null strand as argument.",
+          "property": "nucleotideCounts",
+          "strand": null,
```

Comment:
If `null` is tested as an input, I am thinking it should be an error, rather than to be treated as if `""` was the input. 

(Of course, I imagine languages for which strings are non-nullable will simply skip this test)

This comment must not be read as an endorsement or rejection of the statement "`null` should be tested as an input".

Reply:
EBWODP:

> If `null` is tested as an input, I am thinking it should be an error, rather than to be treated as if "" was the input.

The reason for this belief is that if we consider that the function under test must only accept strings, well `null` is not a string, so this is in error.

Reply:
Yeah, I agree. I had this same thought but had not changed it, yet, because of the ongoing discussion. I will probably change it to expect a throw in an upcoming commit.